src/test/java/selenium/webdriver/SeleniumEasyTest.java

```diff
@@ -0,0 +1,56 @@
+package selenium.webdriver;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.support.ui.Select;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SeleniumEasyTest {
+
+    private static final String URL = "https://demo.seleniumeasy.com/basic-select-dropdown-demo.html";
+
+    private WebDriver driver;
+
+    @BeforeEach
+    public void setup() {
+        ChromeOptions options = new ChromeOptions();
+        options.addArguments("--ignore-ssl-errors=yes", "--ignore-certificate-errors");
+        driver = new ChromeDriver(options);
+    }
+
+    @Test
+    @DisplayName("Test with multiselect")
+    public void multiSelectTest() throws InterruptedException {
+        driver.get(URL);
+        WebElement selectElement = driver.findElement(By.id("multi-select"));
+        Select select = new Select(selectElement);
+        select.selectByIndex(1);
+        select.selectByIndex(3);
+        select.selectByIndex(5);
+        List<WebElement> selectedOptions = select.getAllSelectedOptions();
+        List<String> names = new ArrayList<>();
+        for (WebElement selectedOption : selectedOptions) {
+            String text = selectedOption.getText();
+            names.add(text);
+        }
```

Comment:
I suggest to use Stream api in future , it's a best practice and more modern approach for cases like this, but there's no need to update here as long as everything is fine

Reply:
Stream is used in this task now